src/components/formLabel/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { FormLabel as ChakraFormLabel } from '@chakra-ui/react';
+
+const FormLabel: FC<PropsWithChildren> = ({ children }) => (
```

Comment:
Should we update the `FormControl` component to consume the customised `FormLabel` (I see we don't customise it yet but we might do it in the future and this will avoid diverging the styles...)

Reply:
After checking - we don't want it after all. They serve different purposes. You would use `FormControl` with a form field, whereas you'd use `FormLabel` only in the context of the field group. If we would want to use `FormLabel` here it would require us to allow other usages of our component (and extend the props as a consequence) - I would like to avoid that. Styling-wise they use the same theme so that will ensure a consistent look and feel.

Reply:
I can't follow. In the usage example, they show to use FormLabel inside the form control 
https://chakra-ui.com/docs/components/form-control/usage#usage so why would I use the FormLabel different than the chakra example?

Reply:
Second example there shows closer what I mean